src/V3EmitCFunc.h

```diff
@@ -117,7 +118,7 @@ class EmitCLazyDecls final : public VNVisitorConst {
 class EmitCFunc VL_NOT_FINAL : public EmitCConstInit {
     VMemberMap m_memberMap;
     AstVarRef* m_wideTempRefp = nullptr;  // Variable that _WW macros should be setting
-    int m_labelNum = 0;  // Next label number
+    std::unordered_map<AstJumpBlock*, size_t> m_labelNumbers;  // Label numbers for JumpBlocks
```

Comment:
We should VL_RESTORE and clear this each AstCFunc so all code doesn't massively change on adding a new jump one.

Reply:
Clearing is enough, no need to restore, done.